### ISSUE TITLE
VIC 5 windows 10

### ISSUE THREAD

**Issue by Saadi4469** [UW-Hydro/VIC#895]
Hi,
I am trying to setup VIC 5 on windows 10, but till now I have been struggling to do so. I have downloaded the VIC 5 master from github, but after that I get confused on how to start the process of running VIC. Can anyone help me setup and run VIC 5 on Windows 10?

Cheers,

**Comment by jhamman** [comment]
I recommend taking a look at the VIC documentation. In particular, the FAQ covers why VIC doesn't include Windows support directly: https://vic.readthedocs.io/en/master/FAQ/FAQ/

**Comment by Saadi4469** [comment]
Yes, I downloaded Cygwin but I read somewhere on github that VIC 5 can't be run on Cygwin.

**Comment by Unsolvedakhi** [comment]
I also encounter with same issue, did you try to run VIC 5 on linux?
